api/AmbientLightSensor.json

```diff
@@ -6,24 +6,74 @@
         "support": {
           "chrome": [
             {
-              "version_added": "54"
+              "version_added": "62",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "#enable-generic-sensor",
```

Comment:
This doesn't seem to exist under chrome://flags/ any longer, it was probably removed when generic sensors shipped in 67.

Looking at https://chromium.googlesource.com/chromium/src/+/73373a6ed02e4a2bbd3e42b942cf4c144e79992c%5E%21/#F13 and the current state of [runtime_enabled_features.json5](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/platform/runtime_enabled_features.json5;l=1727;drc=13771712b0eca90b550ee3e9b187b29b83055eca), these bits could always also be enabled by the usual "Experimental Web Platform Features" flag.

Can you use that instead, and collapse these? It doesn't seem worth anyones time to track the more fine-grained flags, and they're more prone to change over time, whereas "Experimental Web Platform Features" has been with us for at least 5 years now.

Reply:
Looks like that flag wasn't removed in Chrome 67, but it was enabled by default, making it redundant to enable manually -- only the #enable-generic-sensor-extra-classes needed to be enabled.  I've double-checked the Experimental Web Platform Features flag and can confirm it enables support since Chrome 56, so I'll swap it out for that flag!